magento/pwa-studio
# [PWA-621] Carry over design tokens from styleguide

## Description

Many of the core values of Venia's new design have been defined in the styleguide. We need to port these to `venia-ui` so we can incorporate these values consistently as we apply the new design. Most of these tokens can be found in `venia-styleguide/src/styles/tokens.css`.

## Related Issue
https://jira.corp.magento.com/browse/PWA-621

## Acceptance 
CSS should use the same patterns as the rest of `venia-ui`:
- Use CSS modules' scoped classes
- Use CSS Grid
- Use CSS custom properties
- Use transitions responsibly
### Verification Stakeholders

@jimbo 
@sirugh 
@dhaecker 
### Specification



### Verification Steps
- Copy `tokens.css` from `venia-styleguide` to `venia-ui` to carry over the CSS custom properties
- Move `index.css` from `venia-concept` into `venia-ui`
- Ensure that `index.css` imports `tokens.css`
- Update `@font-face` in our templates to load `400`, `600`, and `700` weights of `Muli`
- Update `@font-face` in our templates to load the `600` weight of `Source Serif Pro`
- Update existing CSS custom properties to use the new values
- Fix CSS as necessary in places where the new values cause a regression

## Screenshots / Screen Captures (if appropriate)

## Checklist
